admin/src/react-components/service-editor.js

```diff
@@ -325,12 +388,14 @@ class ConfigurationEditor extends Component {
     const displayPath = path.join(" > ");
     return (
       <div key={displayPath} className={this.props.classes.colorInput}>
+        {descriptor.name === "Background Color" && <h3 className="heading-sm mb-24">Nametags</h3>}
```

Comment:
It feels wrong to insert logic like this here and in service-editor (for handling themes). It belongs at a higher level. It would not be difficult to move it up (to where we decide to render the themes section). It just means we won't blindly render the controls for that chunk of config variables.

That being said, I understand this was meant to be done as quickly as possible, and you're still getting familiar with the admin panel.

If this code still exists in 6 months, if there are additional incremental changes planned for the admin panel, or if what I'm saying isn't clear, then we should move this logic to a higher level. If what I'm describing is unclear, then I'm happy to chat through it.